### ISSUE TITLE
Which Stratum Method?

### ISSUE THREAD

**Issue by stone212** [paritytech/parity-ethereum#10045]
_Before filing a new issue, please **provide the following information**._

- **Parity Ethereum version**: 2.1.9
- **Operating system**: Linux
- **Installation**: one-line installer
- **Fully synchronized**: yes
- **Network**: private
- **Restarted**: yes

Which stratum mining method does the beta build-in stratum miner support?  I do not see the info on the wiki: https://wiki.parity.io/Mining

I think the options are:

```
0 - eth-proxy mode (for example, dwarpool.com)
1 - qtminer mode (for example, ethpool.org)
2 - miner-proxy mode (for example, coinotron.com), 3 - nicehash mode
```

(From claymore readme)

It will help to de-bug problems with connecting miner to parity if I can narrow the choice down to one of these.

EDIT: I call this an "Issue" because the issue is that the wiki does not say. :)

EDIT: When I wait longer I see the problem is claymore does not support local stratum proxy.  But anyway I think it is important to have this answer available so I will leave this open.

**Comment by joshua-mir** [comment]
Most mining software (including, presumably Claymore) is designed to connect to a stratum proxy, because that is what pools use as it would likely be unsafe to expose their nodes to the internet publicly. These proxies implement things like authentication (user/worker names, passwords, etc) which parity does not implement, and it will not accept connections attempting to use these features. 

As I understand parity only supports "[plain stratum](https://github.com/ethereum-mining/ethminer/blob/master/docs/POOL_EXAMPLES_ETH.md)"... As you seem to have discovered, there is no equivalent on Claymore. The modes mentioned in the claymore readme are by no means standards (as stratum isn't standardized in the first place), and it might not be worthwhile to document the behavior since it isn't relevant to parity; Ethminer and Genoil's miner are the only miners known to work for certain, which is why they are documented, other miners that respect the scheme/protocol string should also work, but I don't have knowledge of which miners those are. 

Tl;Dr parity implements stratum without any authentication, nor any "proxy mode". The existing documentation shows everything parity supports. It doesn't match the protocol chnages or additional features used by public pools. Documenting which specific non-standard protocols they use should be/is documented by mining software and pools themselves.

**Comment by stone212** [comment]
@joshua-mir 

`parity implements stratum without any authentication, nor any "proxy mode", like pools use as it would likely be unsafe to expose their nodes to the internet publicly. 

If you added this phrase to the wiki it would be helpful.
